webapp/loggedin/modules/analysis/processingChain/processingChainState.js

```diff
@@ -21,9 +22,16 @@ const getProcessingChainOrig = _getStateProp(keys.orig, {})
 
 export const getProcessingChain = _getStateProp(keys.dirty, {})
 
+export const getAttributeUuidsOtherChains = _getStateProp(keys.attributeUuidsOtherChains, [])
+
 // ====== UPDATE
 
-export const assocProcessingChain = chain => R.pipe(R.assoc(keys.orig, chain), R.assoc(keys.dirty, chain))
+export const assocProcessingChain = (chain, attributeUuidsOtherChains) =>
+  R.pipe(
+    R.assoc(keys.orig, chain),
+    R.assoc(keys.dirty, chain),
```

Comment:
can you have one method _assocChain and be called by this method and saveDirty?

Reply:
sure